.travis.yml

```
@@ -21,5 +21,11 @@ before_script:
   - git config --global user.email team+camperbot@freeCodeCamp.org
   - git config --global user.name "CamperBot"
 
-script:
-  - npm run test
+jobs:
+  include:
+    - stage: Test the things
+      script: npm test
+
+    - stage: Trigger build
+      if: branch = master && type != pull_request
+      script: bash trigger-build.sh
```

```suggestion
      script: bash trigger-build.sh
 
```

Newline. UNIX doesn't play well at times.